### ISSUE TITLE
GUI enhancements for long computations

### ISSUE THREAD

**Issue by tpeulen** [Fluorescence-Tools/Olga#11]
Doing long/computations when determining informative FRET pairs in Olga is currently “risky”, as there is no

1. option to open saved FRET efficiency tables for a search of informative pairs

2. “safety net” that prevents accidental closing of OIga

Adding these two options could significantly improve the workflow / user experience.

**Comment by mdimura** [comment]
Yes, that's a long standing problem, that I plan to solve together with a major architecture overhaul. For the time being, there is a hacky hidden option to load Efficiency tables. First, you need to activate `Extras` menu by creating an empty `enableExtras` file in the program folder. Then you can use the `Extras > load results` option. It is deliberately hidden because there are no safety checks if you load a mismatching results file.
